include/sys/bus.h

```diff
@@ -151,16 +151,18 @@ struct bus_driver {
   bus_methods_t bus;
 };
 
-#define BUS_METHODS(dev) ((bus_driver_t *)((dev)->driver))->bus
+#define bus_get_interface(dev)                                                 \
```

Comment:
I am not sure what's correct but it seems that in general we use capital letters to name __#define__ functions
```
~/mimiker$ sum=0; for c in {a..z}; do y=`grep -Ir "#define $c" --exclude-dir=toolchain | wc -l`; sum=$((sum+y)); done; echo $sum
3202
```
```
~/mimiker$ sum=0; for c in {A..Z}; do y=`grep -Ir "#define $c" --exclude-dir=toolchain | wc -l`; sum=$((sum+y)); done; echo $sum
18058
```

It is just a remark

Reply:
There are plenty of non-capital defines as well usually used for macros that behave like functions but are not worth even writing an inline static function. Eg. `bus_write_1`.

Reply:
I agree - it is just a remark